.gitignore

```diff
@@ -13,4 +13,6 @@ tsconfig.tsbuildinfo
 !.yarn/releases
 !.yarn/plugins
 !.yarn/sdks
-!.yarn/versions
\ No newline at end of file
+!.yarn/versions
+
+.env.local
```

Comment:
you should also add `.env`

Reply:
Don't know if we want to have some shared variables in .env. That's why I created my own .env.local. What do you think?